hooks/useCreateSuccess.tsx

```
@@ -0,0 +1,69 @@
+  const chainId = useChainId()
+
+  useEffect(() => {
+    const handleSuccess = async () => {
+      const abi = isExistingContract ? zoraCreator1155ImplABI : zoraCreator1155FactoryImplABI
+      const logs = parseEventLogs({
+        abi,
+        logs: callsStatus?.receipts?.[0]?.logs as any[],
+      }) as any[]
+      if (logs && logs.length > 0) {
+        const relevantLog = isExistingContract ? logs[3] : logs[1]
+        if (relevantLog && relevantLog.args) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)